recipes/compass-interface/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "0.22.1" %}
+
+package:
+  name: compass-interface
+  version: {{ version }}
+
+source:
+  url: https://github.com/the-scouts/compass-interface/releases/download/v{{ version }}/compass-interface-{{ version }}.tar.gz
+  sha256: 3bafb5da507d3b2886a9d56c25ffade9dfb477b4cd6e06b3e259b78f048c0b68
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - compass-interface-core >=0.8.0
+    - pandas
+    - pyarrow
+    - fastapi
+    - conda-forge::python-jose
```

I thought this wasn't supported (though I might be wrong), regardless we only support getting packages from `conda-forge`:
```suggestion
    - python-jose
```